game_mechanics.py

```diff
@@ -404,14 +514,16 @@ def play_othello_game(
     state, reward, done, info = game.reset(verbose)
     # if render:
     #     game.render()
-    sleep(1 / game_speed_multiplier)
+    if verbose:
+        sleep(1 / game_speed_multiplier)
```

Comment:
Why do we want to sleep only when verbose? :thinking: 

Think I'm missing something

Reply:
Because the 'sleep' is only for when you're printing the game out right? There's no need to sleep if you aren't displaying anything